mandatory/1-fix-functions.js

```diff
@@ -33,20 +32,20 @@ function greaterThan10(num) {
 function sortArray(letters) {
   let sortedLetters = letters;
 
-  return sortedLetters;
+  return sortedLetters.sort();
 }
 
 function first5(numbers) {
-  let sliced;
+  let sliced = numbers;
 
-  return sliced;
+  return sliced.slice(0, 5);
 }
```

Comment:
Same as above, the change is correct, but in makes more sense to do it on line 39, `numbers.slice(0,5)`

Reply:
@louisechow 

```
function first5(numbers) {
  let sliced = numbers.slice(0, 5);

  return sliced;
}
```